lighthouse-core/lib/stack-collector.js

```diff
@@ -87,13 +90,15 @@ async function collectStacks(passContext) {
   /** @type {JSLibrary[]} */
   const jsLibraries = await passContext.driver.evaluateAsync(expression);
 
-  return jsLibraries.map(lib => ({
+  const stacks = jsLibraries.map(lib => ({
```

Comment:
*cough* #6515

Reply:
personally I do like giving each object a name instead of `return someDecentAmountOfWork()`.  It also provides the ability to add breakpoint at the end.

but w/e


@connorjclark you requested changes... did you want something in particular?

Reply:
>  you requested changes... did you want something in particular?

Not here, just for the name. LGTM